.gitignore

```diff
@@ -1,6 +1,7 @@
 /doc/_build/vendor
 /doc/_build/output
 /composer.lock
+/phpunit.phar
```

Comment:
This will allow developer to place a PHAR distribution of PHPUnit in the main directory which is picked up and indexed by PHPStorm for code completion.